docs/doc/30-reference/30-sql/10-dml/dml-copy-into-table.md

```diff
@@ -38,19 +38,48 @@ internalStage ::= @<internal_stage_name>[/<path>]
 externalStage ::= @<external_stage_name>[/<path>]
 ```
 
-### externalLocation (for Amazon S3-like)
+### externalLocation
 
-```
-externalLocation (for Amazon S3) ::=
+AWS S3 compatible object storage services:
+
+```sql
+externalLocation ::=
   's3://<bucket>[/<path>]'
-  [ { CREDENTIALS = ( {  { AWS_KEY_ID = '<string>' AWS_SECRET_KEY = '<string>' } } ) } ]
+  CONNECTION = (
+        ENDPOINT_URL = 'https://<endpoint-URL>'
```

Comment:
To support endpoint url starts with `http://`, users need to enable the `allow_insecure`  config in `storage`.